runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java

```diff
@@ -258,6 +266,28 @@ private void handleDeposit(MenuOptionClicked event, Boolean inventory)
 				for (Integer item : items)
 				{
 					tagManager.addTag(item, activeTab.getTag(), false);
+
+					if (config.autoSetQuantity())
+					{
+						int quantity = 0;
+						for (Item containerItem : container.getItems())
+						{
+							if (containerItem == null)
+							{
+								continue;
+							}
+
+							if (containerItem.getId() == item)
+							{
+								quantity += containerItem.getQuantity();
+							}
+						}
+
+						//Don't auto tag items of a quantity of 1
+						if (quantity > 1)
+						tagManager.setTagItemQuantity(activeTab.getTag(), item, String.valueOf(quantity));
```

Comment:
I would probably put this this within `if` brackets to match the code style

Reply:
It's also required to be in brackets